README.md

```diff
@@ -1,41 +1,47 @@
-[![codecov](https://codecov.io/gh/similarweb/finala/branch/master/graph/badge.svg)](https://codecov.io/gh/similarweb/finala)
+4[![codecov](https://codecov.io/gh/similarweb/finala/branch/master/graph/badge.svg)](https://codecov.io/gh/similarweb/finala)
 ![Lint](https://github.com/similarweb/finala/workflows/Lint/badge.svg)
 ![Fmt](https://github.com/similarweb/finala/workflows/Fmt/badge.svg)
+[![Go Report Card](https://goreportcard.com/badge/github.com/similarweb/finala)](https://goreportcard.com/report/github.com/similarweb/finala)
 
+<p align="center">
+    <img src="https://raw.githubusercontent.com/similarweb/finala/docs/update-readme-md/docs/images/logo.png" width="400">
+</p>
+
+---
 
 # Finala
 
 A resource cloud scanner that analyzes and reports about wasteful and unused resources to cut unwanted expenses.
 The tool is based on yaml definitions (no code), by default configuration OR given yaml file and the report output will be saved in a given storage.
-Currently we're not taking into account reservations, private pricing, etc. (WIP)
+Currently we're not taking into account reservations, private pricing, etc.
 
 ## Supported Services
-
 AWS:
-* RDS
-* EC2 (ELB, ALB, EBS)
-* DynamoDB
-* ElasticCache
-* DocumentDB
-* IAM user last activity
-* Lambda
-* Neptune
-* Kinesis
-* RedShift
-
-More to come...
+Resource            | Sub Resources|
```

Comment:
Nice changed.
I think we can improve this table by adding 2 more column:

1. Description: with more explanation of collector metrics 
2. Direct link to the configuration file in Finala project and also in https://github.com/similarweb/finala-helm repo

WDYT?

Reply:
changed.